packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -96,16 +100,67 @@ class StoryOfYourFlexWidget extends StatefulWidget {
   _StoryOfYourFlexWidgetState createState() => _StoryOfYourFlexWidgetState();
 }
 
-class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
+class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
+    with TickerProviderStateMixin {
+  @override
+  void initState() {
+    super.initState();
+    entranceController = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 500),
```

Comment:
nit: make the duration into const maybe?

Reply:
It is const.

Reply:
Sorry, I mean, extract it into top level const?